arraycontext/fake_numpy.py

```diff
@@ -160,15 +162,16 @@ def loopy_implemented_elwise_func(*args):
         if name in self._c_to_numpy_arc_functions:
             from warnings import warn
             warn(f"'{name}' in ArrayContext.np is deprecated. "
-                    "Use '{c_to_numpy_arc_functions[name]}' as in numpy. "
+                    f"Use '{self._c_to_numpy_arc_functions[name]}' as in numpy. "
                     "The old name will stop working in 2021.",
                     DeprecationWarning, stacklevel=3)
 
         # normalize to C names anyway
         c_name = self._numpy_to_c_arc_functions.get(name, name)
 
         # limit which functions we try to hand off to loopy
-        if name in self._numpy_math_functions:
+        if (name in self._numpy_math_functions
+                or name in self._c_to_numpy_arc_functions):
```

Comment:
Not quite sure if this was a bug or intentially skipped?

As far as I can tell, before calling `actx.np.atan` would raise the warning above and then say the `atan` is not implemented.

Reply:
No, I think this was just overlooked and never actually used. Thanks for spotting (and testing!) it.